web/static/web/css/header.css

```diff
@@ -1,144 +1,332 @@
+    #header .logo .limg.medium {
+        display: block;
+    }
+
+    #header .logo .limg.small {
         display: none;
     }
+}
 
-    .limg.medium {
-        display: inline;
+/* Narrower space between elements in navbar */
+@media screen and (max-width: 1100px) {
+    #header {
+        --logo-padding: 1em;
+        --nav-space-between: 1em;
     }
 }
 
-@media screen and (max-width: 730px) {
-    .limg.large {
+/* Switch to mobile layout with burger */
+@media screen and (max-width: 991px) {
+    #header {
+        --burger-space: minmax(calc(4em + var(--burger-margin)), 1fr);
+
+        grid-template-columns: var(--burger-space) auto var(--burger-space);
+        grid-template-rows: var(--header-height);
+    }
+
+    #header.active {
+        height: auto;
+    }
+
+    #header div.logo-centering {
         display: inline;
     }
 
-    .limg.small {
+    #header .logo {
+        grid-template-columns: 1em var(--logo-space) 1em;
+    }
+
+    #header .logo .limg.large {
+        display: block;
+    }
+
+    #header .logo .limg.medium {
         display: none;
     }
 
-    .limg.medium {
+    #header .logo .limg.small {
         display: none;
     }
 
-    .nav {
-        grid-template-columns: 1fr 3fr;
+    #header #burger {
+        display: inline;
     }
 
-    .nav .logo {
-        margin-right: 80px;
+    #header #nav {
+        grid-row: 2;
+        grid-column-start: 1;
+        grid-column-end: 4;
+        margin: 0;
     }
 
-    .nav .menu {
+    #header #nav,
+    #header #side-nav {
         display: none;
-        grid-row-start: 2;
-        grid-column-start: 1;
-        grid-column-end: 3;
     }
 
-    .nav.active .menu {
+    #header.active #nav,
+    #header.active #side-nav {
         display: inline;
-        padding-left: 10px;
-        padding-bottom: 20px !important;
+    }
 
+    #header #nav .nav-item,
+    #header #side-nav .nav-item {
+        margin: 0;
     }
 
-    .nav.active {
-        height: auto;
+    #header #side-nav {
+        align-self: end;
+        justify-self: center;
+        grid-row-start: 3;
+        grid-column-start: 1;
+        grid-column-end: 5;
+        margin-top: 2em;
+        margin-bottom: 1em;
     }
 
-    .burger {
-        display: inline;
+    #header.active #side-nav #nav-user-dropdown .menu {
+        /* Fixes Semantic UI de-centering the dropdown menu */
+        left: initial;
+    }
+
+    #header #side-nav .lang-link {
+        position: static;
     }
 }
 
+/* Switch to medium logo */
 @media screen and (max-width: 600px) {
-    .limg.large {
+    #header .logo .limg.large {
         display: none;
     }
 
-    .limg.small {
-        display: none;
+    #header .logo .limg.medium {
+        display: block;
     }
 
-    .limg.medium {
-        display: inline;
+    #header .logo .limg.small {
+        display: none;
     }
 }
 
+/* Switch to small logo */
 @media screen and (max-width: 340px) {
-    .limg.medium {
+    #header .logo .limg.large {
```

Comment:
For all of these logo selectors, it would probably be easier to specify that the `display` property should be set to `block` in intervals, rather than having to use specify `none` on all of the others.

Reply:
Not really sure what you mean by "set to `block` in intervals".. Specifying `none` for the other image sizes is purely for readability, because then you get the same format wherever the size is changed.

Reply:
Very good idea! Made it much more readable :)